index.html

```diff
@@ -5,6 +5,6 @@
     <title>Title</title>
 </head>
 <body>
-
+This is the home page of the website.
```

Comment:
Make **HEADER** and add nav bar and search in the header.

Reply:
I have added. Please review it.